Revising notebook 1

AllenDowney opened the pull request:
Adding text and moving toward compliance with the style guide.

AllenDowney committed:
Revising notebook 1

AllenDowney commented:
@AlexanderFengler This is a work in progress, but there are some style changes we might want to sync on before revising the rest of the notebooks.

AlexanderFengler commented:
@AllenDowney this basically looks good.
Will try to take the implicit style-guide and start working on the notebook 2.

AllenDowney committed:
Adding text for the second half

AllenDowney commented:
<p>That makes sense, but I am trying to simplify the example so we introduce the minimum number of new concepts at each step.</p><p> </p>

---

 <a href='https://app.reviewnb.com/AllenDowney/SurveyDataPyMC/pull/2/discussion/#comment-2770819278'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

AllenDowney commented:
<p>Got it!</p>

---

 <a href='https://app.reviewnb.com/AllenDowney/SurveyDataPyMC/pull/2/discussion/#comment-2770819294'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

AllenDowney commented:
@AlexanderFengler I added text to the second half of the notebook, and made some additional code changes, mostly trying to simplify things for purposes of explanation. I'm thinking about what it will take to explain all of this, and it's a lot!

Also, I removed the discussion of the posterior predictive distribution for now, because the predictions we generate in this notebook are deterministics that depend on the posterior, not technically posterior predictives.

So please take another look, especially at the second half, and let me know if you have thoughts or suggestions. Thank you!
